pytx/pytx/batch.py

```diff
@@ -0,0 +1,85 @@
+import json
+
+from access_token import get_access_token
+from request import Broker
+
+from vocabulary import Batch as b
+from vocabulary import ThreatExchange as t
+from errors import (
+    pytxFetchError
+)
+
+
+class Batch(object):
+
+    """
+    Class for making Batch requests to the API.
+    """
+
+    def __init__(self, **kwargs):
+        """
+        Initialized the object.
+        """
+
+    @classmethod
```

Comment:
I notice in a few places the use of @classmethod instead of @staticmethod. It doesn't appear that we use cls in this method's logic - should we make this and other similar instances @staticmethod?